static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -0,0 +1,403 @@
+    }
+}
+
+.newsletter-card__header {
+
+}
+
+.newsletter-card__imagewrapper {
+    background-color: $guardian-brand;
+    height: 0;
+    padding-bottom: 60%;
+
+    svg {
+        display: block;
+        margin: auto;
+        display: block;
+        margin: auto;
+        padding-top: 10%;
+        height: 80%;
+        width: 40%;
+    }
+}
+
+.newsletter-image {
+    position: relative;
+}
+
+.newsletter-card__name {
+    @include fs-header(5);
+    font-weight: 900;
+    padding-top: $gs-baseline / 2;
+    padding-bottom: $gs-baseline / 2;
+}
+
+.newsletter-card__content {
+    @include fs-textSans(3);
+    line-height: 18px;
+    background-color: $comment-support-2;
+    height: 9em;
+    padding: 0 $gs-gutter / 2 0 $gs-gutter / 2;
+
+
+    @include mq(tablet) {
+        height: 12em;
+    }
+
+    @include mq(desktop) {
+        height: 12em;
+    }
+}
+
+.newsletter-card__meta {
+    background-color: $comment-support-2;
+    border-bottom: 1px solid $neutral-3;
+    padding: $gs-gutter / 2;
+    padding-bottom: $gs-baseline * 2;
+    height: gs-height(1);
+
+    * {
+        -webkit-box-sizing: border-box;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
+    }
+
+
+    .newsletter-card__text-input {
+        @include fs-textSans(2);
+        -webkit-border-radius: 62.5rem 0 0 62.5rem;
+        border-radius: 62.5rem 0 0 62.5rem;
+        border: 1px solid $neutral-3;
+        border-right: 0;
+        float: left;
+        padding-left: $gs-baseline;
+        width: 60%;
+        height: 3em;
+    }
+
+    .newsletter-card__lozenge {
+        @include fs-textSans(2);
+        color: #ffffff;
+        border: 1px solid $news-main-1;
+        background-color: $news-main-1;
+        -webkit-border-radius: 62.5rem;
+        border-radius: 62.5rem;
+        border-width: 1px;
+        height: 3em;
+        width: 48%;
+        margin: 0;
+        padding: 0;
+
+        &.lozenge--is-updating {
+            display: inherit;
+            font-size: 0;
+            height: 2.43rem;
+            background-color: $neutral-5;
+            border: 1px solid $neutral-3;
+            background-repeat: no-repeat;
+            background-position: center;
+            background-size: 40px;
+        }
+    }
+
+
+
+
+    .newsletter-card__lozenge--submit {
+        -webkit-border-radius: 0 62.5rem 62.5rem 0;
+        border-radius: 0 62.5rem 62.5rem 0;
+        border-left: 0;
+        margin-right: 0;
+        width: 40%;
+    }
+
+    .newsletter-card__lozenge--preview {
+        color: $news-main-1;
+        background-color: transparent;
+        float: right;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: darken($comment-support-2, 10%);
+        }
+    }
+
+    .inline-arrow-right {
+        fill: $news-main-1;
+        position: absolute;
+        height: 2rem;
+        line-height: 2rem;
+    }
+}
+
+.newsletter-card__preview {
+    line-height: 2rem;
```

Comment:
Oh, I remember this! `newsletter-card__preview` is helping centre a SVG and align it with the text. For some reason `px` doesn't have any effect, hence the `rem` here. 😐🔫 
Will remove the magic numbers from elsewhere though!